xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -2,34 +2,51 @@
 // Edit
 // --------------------------------------------------
 
-
-.bottombuttons {
+.sticky-buttons {
   background-color: @xwiki-page-content-bg;
   .buttons {
     margin: 0;
     padding: 1em;
   }
 
-  &.bottombuttons-fixed {
+  &.sticky-buttons-fixed {
     z-index: 100;
     position: fixed;
     bottom: 0;
     background-color: @breadcrumb-bg;
+    @media (max-width: @screen-sm-max) {
+      #commentinput, #autosaveControl, .metamodifs {
+        display: none;
+      }
+    }
+  }
+
+  input[type=checkbox] {
+    vertical-align: middle;
+    position: relative;
+    margin: 0;
+    margin-left: 1em;
+    bottom: 1px;
```

Comment:
this value sounds strange

Reply:
I would rather remove it but since you wanted to have the checkbox aligned with the input, there wasn't many other choices from what I saw. What do you think we can do about that?

Reply:
I don't know how it looks exactly, but one thing to check would be to see the value of margin-bottom or padding-bottom. Maybe somewhere is 6px or another value that can be reused. Setting a bottom especially of this size sounds strange. 
If you really need bottom and we don't have a param for 1px, you can have it like this, but just check if it's really needed.

Reply:
@evalica It looks like this by default:
![image](https://user-images.githubusercontent.com/2213999/41232255-10694f40-6d86-11e8-8d56-69ba936e7700.png)